package.json

```diff
@@ -11,8 +11,5 @@
     "prettier": "^2.3.2",
     "semantic-release": "^19.0.0",
     "semver": "^7.3.5"
-  },
-  "dependencies": {
-    "@outreach/semantic-release-circleci-orb": "^1.1.3"
```

Comment:
We need this

Reply:
Ah, yeah, so we do need a block here, but `package.json` doesn't actually support blocks. I want to `fromJSON` this and keep user's changes by basically doing a merge but I haven't gotten around to it yet :(

Reply:
Oh yeah... I forgot about that json annoyance. Oh well, I just added it back as a short term fix. Maybe we can get the `fromJSON` loading working by next release.